Xwt.XamMac/Xwt.Mac/ViewBackend.cs

```diff
@@ -241,6 +243,8 @@ public void SetCursor (CursorType cursor)
 				Cursor = NSCursor.ArrowCursor;
 			else if (cursor == CursorType.Move)
 				Cursor = NSCursor.ClosedHandCursor;
+			else if (cursor == CursorType.Drag)
```

Comment:
I've double checked the default Finder behaviour on Mac and `NSCursor.DragCopyCursor` seems to be wrong. The default drag operation has no special cursor (defaults to `NSCursor.ArrowCursor`), only if you press the Alt key it will change to the `NSCursor.DragCopyCursor`.

Reply:
I renamed the cursor type `Drag` to `DragCopy`.
Also added `NotAllowed` cursor type.